test/test.js

```diff
@@ -13,8 +13,10 @@ var Primus = require('primus')
 function client(srv, primus, options){
   var addr = srv.address() || {}
     , address = addr.address || 'localhost'
-    , port = addr.port || 8080
-    , url = 'http://' + address + ':' + port;
+    , port = addr.port || 8080;
+  // Node 0.12 IPv6 fix
+  if (address === '::') address = 'localhost';
+  var url = 'http://' + address + ':' + port;
```

Comment:
Maybe there is no point on getting the `address` from `srv.address()`, we only use `localhost` anyway.
I would just rewrite this as follow:
```js
var addr = srv.address() || {}
  , port = addr.port || 8080
  , url = 'http://localhost:'+ port;
```